### ISSUE TITLE
Некорректное значение текущего тарифа (МО)

### ISSUE THREAD

**Issue by avbor** [alryaz/hass-lkcomu-interrao#39]
Начиная с 1.07.2024 в Московской обласити применяются тарифы с разной стоимостью кВт\ч в зависимости от суммарного потребелния в месяц - https://www.mosenergosbyt.ru/individuals/tariffs-n-payments/tariffs-mo/
![image](https://github.com/user-attachments/assets/e323991a-5915-4ed7-851f-922aa683bcaa)
![image](https://github.com/user-attachments/assets/9abea819-e3f3-431a-9c77-0a2f11bd92a6)
![image](https://github.com/user-attachments/assets/8d3d3d93-4d60-499f-93ca-e7a33f72fda8)

При этом компонет некорректно отображает стоимость по тарифу (отображается максимальная стоимость по диапазону потребления 3):
![image](https://github.com/user-attachments/assets/bfe4af18-5da7-4cdc-9199-ea16ae510f10)

Я так понимаю компонент уже заброшен, но вдруг случится чудо и автор внесет правки...
@alryaz очень ждем фиксов =)

**Comment by diemon24** [comment]
Да, я тоже испытываю сложности с получением стоимостей тарифов ночь/день. Видимо, придется самому откуда-то тянуть. Вручную вносить очень не хочется...
